### DISCUSSION TITLE
RTL support?

### DISCUSSION THREAD

**Discussion by sanketsahu** [gluestack/gluestack-ui#506] [Feature Request]
Hi, I was wondering if we have the support of RTL for supporting languages like Arabic and Urdu.

**Comment by JayJayNinetyFive** [comment]
No RTL support yet? 😭😭😭

**Reply by PPHelios** [reply]
Is there any plan to support RTL?

**Reply by JayJayNinetyFive** [reply]
waiting for this :(

**Reply by Viraj-10** [reply]
Hey @JayJayNinetyFive, All of our components have RTL support. If you are facing any issue regarding this please create a issue on repo.

**Comment by sanketsahu** [comment]
It looks like all the components have RTL support now.
